### PR TITLE
Add Hardware#total_cores based on MiqSystem.num_cpus

### PR DESCRIPTION
**Pull request by bdunne**
https://github.com/ManageIQ/manageiq/blob/master/gems/pending/util/miq-system.rb#L48:L62

### CODE REVIEW

**Comment by bdunne** [comment]
@jrafanie @Fryguy Please review.

We saw the incorrect core count yesterday in our debugging.

File: lib/linux_admin/hardware.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+module LinuxAdmin
+  class Hardware
+    def num_cpus
```
Comment (jrafanie) [inline lib/linux_admin/hardware.rb]: Maybe `total_cores`

**Comment by bdunne** [comment]
@jrafanie Updated

**Comment by jrafanie** [comment]
Looks good.

**Pull request merged**